tbdex/validator/validator_test.go

```diff
@@ -0,0 +1,6 @@
+package validator_test
+
+// func TestValidate_Invalid(t *testing.T) {
+// 	err := tbdex.Validate(tbdex.TypeResource, []byte(`{"foo": "bar"}`))
+// 	assert.Error(t, err)
+// }
```

Comment:
is this TODO in this PR or a follow up?

Reply:
https://github.com/TBD54566975/tbdex-go/commit/8622cccbcbf1cac735c111df1dea32e5483bdd58